deploy.sh

```diff
@@ -64,7 +64,7 @@ if [ -z "$INPUT_ARCHIVE" ]; then
         cd "$DIR_TO_ZIP";
     fi
 
-    zip -r --quiet "$ZIP_FILENAME" . -x "@$EXCLUSION_FILE"
+    zip "$INPUT_CUSTOM_ZIP_FLAGS" -r --quiet "$ZIP_FILENAME" . -x "@$EXCLUSION_FILE"
```

Comment:
Won't wrapping this in quotes limit us to one flag?  If so, should that be noted in the description or renamed to `...ZIP_FLAG`?

Reply:
I don't believe so. I thought quotes just prevented expansion and globbing, so its preserved as-is.

Reply:
```
[ec2-user@ip-172-xx-xx-67 ~]$ echo $ZIP_FLAGS
--freshen --symlink
[ec2-user@ip-172-xx-xx-67 ~]$ zip $ZIP_FLAGS -r test.zip me
[ec2-user@ip-172-xx-xx-67 ~]$ unzip -l test.zip
Archive:  test.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
        5  11-02-2021 19:13   me
---------                     -------
        5                     1 file
[ec2-user@ip-172-xx-xx-67 ~]$ zip "$ZIP_FLAGS" -r test.zip me

zip error: Invalid command arguments (long option 'freshen --symlink' not supported)
[ec2-user@ip-172-xx-xx-67 ~]$ 
```

confirmed thanks. @ToothlessRebel - we need to rethink.

Reply:
I believe it's possible to pass multiple arguments, just have to research it a bit.

Reply:
I think I could easily remove quotes and it would work, but then shellcheck screams at me. Want to find best of both worlds of a solution that doesn't require suppressing linting rules, but still works to allow n variables.

cc @Bankamah

Reply:
I just removed the quotes. Ran into a linter error (https://github.com/koalaman/shellcheck/wiki/SC2086) and attempted to push variables into an array and explode, but it was very finicky.

I'm thinking since these parameters are provided by the build script, even while we've opened up some possibilities of abuse. It would be self-inflicted, I think this is okay.